client/lib/analytics/fullstory.js

```diff
@@ -42,6 +42,14 @@ export function recordFullStoryEvent( name, _props ) {
 }
 
 function maybeAddFullStoryScript() {
+	// Don't record user activity on Checkout pages
+	if ( document.location.href.indexOf( 'checkout' ) !== -1 ) {
+		if ( window.FS ) {
+			window.FS.shutdown();
```

Comment:
After digging in a bit more, I'm not sure if this will be sufficient.

I've tested a session and it seems like the checkout portion is still captured. As noted in the FullStory docs:

> Because there is likely to be a delay between when the FullStory script is loaded and when you call `FS.shutdown()`, you don't want to rely on shutdown/restart to exclude data from FullStory. We recommend only relying on [excluded elements](https://help.fullstory.com/hc/en-us/articles/360020623574) for keeping sensitive data out of FullStory.

I will link a recorded session example in Slack.

Per the recommendation to exclude elements, we have a couple of options - add the `.fs-exclude` class to the checkout component or add the `.composite-checkout` class to the exclude list in the FullStory admin.

I tested the latter and it worked like a charm, completely blocking out the cart/checkout experience from view.

I think it would be fine to still implement this `FS.shutdown` approach as well but it doesn't seem like we want to rely on it alone. That being said, If we're going to run `FS.shutdown()` will we then need to run `FS.restart()` at some point afterwards? I haven't been able to successfully test a flow where I go into the cart/checkout and the recording stops so I can't confirm if it resumes recording when leaving the checkout view.